docs/development/authentication.md

```diff
@@ -42,6 +42,25 @@ details worth understanding:
   password in the development environment, those passwords won't
   work.  It also prints out the user's **current** API key.
 
+### Apple
+
+* Visit https://developer.apple.com/account/resources/,
+  [Create a Sign In Certificate](https://help.apple.com/developer-account/?lang=en#/dev77c875b7e)
+
+* [Create a Services ID](https://help.apple.com/developer-account/?lang=en#/dev1c0e25352). Fill the redirect URL with
+  `http://zulipdev.com:9991/complete/apple/` .
```

Comment:
This comes before creating the private key.

Here's a handy banner Apple shows when you do any one of these steps (as they apparently realize that the process is too complicated and underdocumented):
![image](https://user-images.githubusercontent.com/28173/79810647-62ef4580-8328-11ea-8c6f-04fb343df40e.png)

Reply:
```suggestion
* [Create a Services ID](https://help.apple.com/developer-account/?lang=en#/dev1c0e25352). When prompted for a "Return URL", enter
  `http://zulipdev.com:9991/complete/apple/` .
```

Based on the text in their UI here:
![image](https://user-images.githubusercontent.com/28173/79810838-e3ae4180-8328-11ea-9bc7-17239138a502.png)

Reply:
made the changes. do we want to make "Services ID", "Service ID"? In the apple banner screenshot they've written "Service ID"

Reply:
Well spotted! :smile: 

They seem to be not quite consistent about it. The docs we're linking to all say "Services ID" with the plural, though -- or at least all the places I remember spotting did. They also say it that way in the part of the UI where you actually go to create one:
![image](https://user-images.githubusercontent.com/28173/79915773-6e03ad80-83dc-11ea-98de-ec2dc538be00.png)
So I think that's the version to stick to, for the least possible amount of inconsistency.

Reply:
okay, thanks for clarifying. I really like that you share screenshots for most things to make it clear.